README.ja.md

```
@@ -0,0 +1,605 @@
+### skill の発見 — 2 つのスコープ
+
+| スコープ    | 場所                                   | 意味                                                                                                   |
+| ----------- | -------------------------------------- | ------------------------------------------------------------------------------------------------------ |
+| **User**    | `~/.claude/skills/<name>/SKILL.md`     | 個人用の skill。Claude CLI で開くあらゆるプロジェクト間で共有されます。                                |
+| **Project** | `~/mulmoclaude/.claude/skills/<name>/` | MulmoClaude ワークスペーススコープの skill。名前が user と衝突した場合、project が **優先** されます。 |
+
+フェーズ 0 では両スコープとも読み取り専用です — 編集はファイルシステム上で行います。将来のリリースで MulmoClaude 自身が project スコープの skill を作成 / 編集できるようになります。
+
+### Docker サンドボックス vs 非 Docker
+
+MulmoClaude のデフォルトの **Docker サンドボックスモード** は安全性のため Claude Code をコンテナ内に隔離します ([セキュリティ](#security) を参照)。skill の動作は 2 つのモード間で異なります:
```

markdownlint-cli2 (0.22.0)

[warning] Link fragments should be valid

(MD051, link-fragments)